app/models/form.rb

```diff
@@ -71,7 +71,6 @@ def self.receivable_association
   def self.terminate_sub_relationships(form_ids)
     Form.where(id: form_ids).update_all(original_id: nil)
     FormVersion.where(form_id: form_ids).delete_all
-    Questioning.where(form_id: form_ids).delete_all
```

Comment:
Why this change?

Reply:
Specs pass without it. Questionings get deleted by Mission directly. This code has a lot of quirks, just shaving off a few.